Islandora/islandora_badges
# Altmetrics badge open in new window

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2223)

# What does this Pull Request do?

Altmetric badge link now opens in a new window instead of current window.

# What's new?
That's it.

# How should this be tested?

- Visit an object with an Altmetric badge
- Follow the link by clicking on the badge
- Observe how you are taken in your current window to the Altmetric site
- Check out this branch
- Do the same
- Observe that your link now opens in a new tab

# Interested parties
@rosiel  @Islandora/7-x-1-x-committers
